Misaki14987/.env

```
@@ -0,0 +1 @@
+PUBLIC_DOUABO_API_KEY='bf7a0525-554f-4912-9c9c-439c325e600b'
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)